srcpkgs/darkhttpd/template

```diff
@@ -1,19 +1,25 @@
 # Template file for 'darkhttpd'
 pkgname=darkhttpd
 version=1.12
-revision=1
-short_desc="A small and secure static webserver"
+revision=2
+short_desc="Small and secure static webserver"
 maintainer="Andrea Brancaleoni <abc@pompel.me>"
-license="BSD"
-homepage="http://dmr.ath.cx/net/darkhttpd/"
-distfiles="$homepage/$pkgname-$version.tar.bz2"
+license="ISC"
+homepage="https://unix4lyfe.org/darkhttpd/"
+distfiles="https://unix4lyfe.org/darkhttpd/darkhttpd-${version}.tar.bz2"
 checksum=a50417b622b32b5f421b3132cb94ebeff04f02c5fb87fba2e31147d23de50505
 CFLAGS="$CFLAGS $LDFLAGS"
 
+# Create darkhttpd system user/group
+system_accounts="${pkgname}"
+make_dirs="/srv/www/${pkgname} 0755 ${pkgname} ${pkgname}"
+
```

Comment:
i couldn't find any style guidelines for system/user account creation, so i don't know if their position within the template is correct.